### PR TITLE
Feature/xivy 5005 jenkins build

### PR DESCRIPTION
**Pull request by ivy-rew**
finally a full @IvyProcessTest chain that verifies that our UiPathDemo runs as expected.

### CODE REVIEW

File: market/uipath/demo/Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+      defaultValue: 'https://jenkins.ivyteam.io/job/ivy-core_product/job/master/lastSuccessfulBuild/')
+  }
+
+  stages {
+    stage('build') {
+      steps {
+          script {
+            def mavenParameters = "-Divy.engine.list.url=${params.engineListUrl} " +
+              "-Divy.engine.version=[9.2.0,] -Dmaven.test.failure.ignore=true " +
+              "-f market/uipath/demo/pom.xml"
```
Comment (ivy-rew) [inline market/uipath/demo/Jenkinsfile:28]: @alexsuter  maybe you know a nicer solution to this? I'd prefer to set another work.dir for the whole build chain ... resting where the Jenkinsfile is. But didn't quickly see who to do it.

Comment (ivy-rew) [reply market/uipath/demo/Jenkinsfile:28]: sample build: https://jenkins.ivyteam.io/job/market-connector-uipath/job/feature%252FXIVY-5005-jenkins-build/

Comment (alexsuter) [reply market/uipath/demo/Jenkinsfile:28]: you can use 
dir('my-working-dir') {
    // do what ever i like
}

File: market/uipath/demo/Jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,41 @@
+pipeline {
+  agent {
+    docker {
+      image 'axonivy/build-container:web-1.0'
```
Comment (alexsuter) [inline market/uipath/demo/Jenkinsfile:4]: I don't like this container. But this is another story.

File: market/uipath/demo/Jenkinsfile [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+  stages {
+    stage('build') {
+      steps {
+          script {
+            def mavenParameters = "-Divy.engine.list.url=${params.engineListUrl} " +
+              "-Divy.engine.version=[9.2.0,] -Dmaven.test.failure.ignore=true " +
+              "-f market/uipath/demo/pom.xml"
+
+            maven cmd: "clean install " + mavenParameters
```
Comment (alexsuter) [inline market/uipath/demo/Jenkinsfile:30]: verify shoud be enough?
install only creates unecessary IO

**Review by alexsuter — CHANGES_REQUESTED** [review]

File: market/uipath/demo/Jenkinsfile (hunk 2 above)
Comment (ivy-rew) [reply market/uipath/demo/Jenkinsfile:4]: I only need @IvyProcessTEst support... is there a better one? ... just copied blindly from 'project-build-examples' :+1:

**Review by alexsuter — APPROVED** [review]

**Review by alexsuter — APPROVED** [review]

**Pull request merged**